qemu/tests/cfg/cpuinfo_query.cfg

```diff
@@ -11,10 +12,13 @@
             query_cmd = " -M ?"
             cpu_info = "pc,rhel6.4.0,rhel6.3.0,rhel6.2.0,rhel6.1.0,rhel6.0.0,rhel5.5.0,rhel5.4.4,rhel5.4.0"
         - qcpu_dump:
-            no Host_RHEL.5
+            only Host_RHEL.6
             query_cmd = " -cpu ?dump"
             cpu_info = "Opteron_G4,Opteron_G3,Opteron_G2,Opteron_G1,SandyBridge,Westmere,Nehalem,Penryn,Conroe"
         - qcpu_model:
-            no Host_RHEL.5
+            only Host_RHEL.6
             query_cmd = " -cpu ?model"
             cpu_info = "Opteron_G4,Opteron_G3,Opteron_G2,Opteron_G1,SandyBridge,Westmere,Nehalem,Penryn,Conroe"
+        - qcpu_info:
+            query_cmd = " -cpu ?"
+            cpu_info = "Opteron_G4,Opteron_G3,Opteron_G2,Opteron_G1,SandyBridge,Westmere,Nehalem,Penryn,Conroe"
```

Comment:
the scripts works fine on latest RHEL6.7 host.
Do we still need to care about old versions RHEL6.x ? I can modify if it is needed.

Mike